src/components/common/Button/index.js

```diff
@@ -19,7 +19,7 @@ const Button = ({ testID, handleOnPress, additionalStyles, title, isLoading }) =
 Button.propTypes = {
   testID: string,
   handleOnPress: func.isRequired,
-  additionalStyles: object.isRequired,
+  additionalStyles: oneOfType([arrayOf(object), object]),
```

Comment:
```suggestion
  additionalStyles: oneOfType([arrayOf(ViewPropTypes.style), ViewPropTypes.style]),
```

You can import `ViewPropTypes` from `reat-native`.
I think that just this should work too, not sure though.
```suggestion
  additionalStyles: ViewPropTypes.style,
```